tsfile/src/main/java/org/apache/iotdb/tsfile/utils/ReadWriteIOUtils.java

```diff
@@ -326,6 +326,15 @@ public static int write(ByteBuffer byteBuffer, OutputStream outputStream) throws
     return len;
   }
 
+  /**
```

Comment:
How about merge this class and `SerializeUtils.java `

Reply:
Not necessary and burdensome. This one is mainly for primitive types or java built-in classes, and SerializeUtils is mainly for our customized classes.